OracleWebLogic/samples/1221-multihost/bootstrap.sh

```diff
@@ -42,9 +42,19 @@ docker network create --driver overlay $network
 
 # Save existing defined image to a file to be loaded later into the registry created above
 eval "$(docker-machine env -u)"
+docker save $adminimage > _tmp_docker.img 
+
+# Load, tag, and publish the admin server image servlet-proxy in setenv.sh
+eval "$(docker-machine env $orchestrator)"
+docker load -i _tmp_docker.img && rm _tmp_docker.img
+docker tag $adminimage 127.0.0.1:5000/$adminimage
+docker push 127.0.0.1:5000/$adminimage
+
+# Save existing defined image to a file to be loaded later into the registry created above
+eval "$(docker-machine env -u)"
 docker save $image > _tmp_docker.img 
 
-# Load, tag, and publish the image set in setenv.sh
+# Load, tag, and publish the managed server image app-deploy set in setenv.sh
```

Comment:
I changed setenv.sh to have two images an $admin-image which represents the 1221-servlet-proxy (HttpProxyServlet) and $image which represents the 1221-appdeploy (sample app).
The 1221-servlet-proxy image is created by extending the 1221-appdeploy image. The HttpServletProxy (1221-servlet-proxy) application gets deployed to the admin server. The HttpServletProxy will route calls to applications deployed on the managed servers running in a WLS Cluster. The sample (1221-appdeploy) application gets deployed to servers in the WLS cluster DockerCluster.
When you make an HTTP call to an application using the admin server address (http://localhost:8001/sample) the HttpServletProxy will route the call to servers (server1 or server2) running in the WLS cluster.